leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -345,7 +345,10 @@ private void proposeLeadership(LeadershipToken token) {
             proposer.propose(seq, null);
         } catch (PaxosRoundFailureException e) {
             // We have failed trying to become the leader.
-            leaderLog.warn("Leadership was not gained", e);
+            leaderLog.warn("Leadership was not gained because: " + e.getMessage() + ". "
```

Comment:
I wouldn't put the exception in the middle, you probably want the full exception being passed as the exception argument at the end, then you'll get the full stacktrace.